Dockerfile.microos.dapper

```diff
@@ -0,0 +1,11 @@
+FROM opensuse/tumbleweed
```

Comment:
I will pin this before moving out of draft (there are no tags other than latest atm)